src/main/java/net/minecraftforge/common/extensions/IForgeTagBuilder.java

```diff
@@ -67,4 +67,82 @@
     {
         return getBuilder().add(new ITag.OptionalTagEntry(location));
     }
+    
+    /**
+     * Adds a registry entry to the tag json's remove list. Callable during datageneration.
+     * @param element The entry to remove
+     * @return The builder for chaining
+     */
+    default TagsProvider.Builder<T> removeRegistryEntry(final IForgeRegistryEntry<T> entry)
```

Comment:
Use same naming scheme as in 1.17 to reduce upgrading or downgrading complexity.